code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -234,12 +264,20 @@
   * If the target is on the same tile as the plant, destroy the vine
   * Removes any QDELETED vines from the vines list.
   */
+
 /mob/living/simple_animal/hostile/venus_human_trap/proc/pull_vines()
 	for(var/datum/beam/B in vines)
 		if(istype(B.target, /atom/movable))
 			var/atom/movable/AM = B.target
 			if(!AM.anchored)
 				step(AM, get_dir(AM,src))
+
+		if(iscarbon(B.target)) // If they dont get away quickly, make them take constant stamina damage
+
+			var/mob/living/L = B.target
+			L.apply_damage(5, STAMINA, BODY_ZONE_CHEST)
```

Comment:
```suggestion
			L.apply_damage(20, STAMINA, BODY_ZONE_CHEST)
```

With players possible to easily rescue this needs to be higher so that they are less able to crawl away without help